sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/PartitionKeyInternalTest.java

```diff
@@ -4,6 +4,7 @@
 package com.azure.cosmos.implementation.directconnectivity;
 
 import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.implementation.routing.HexConvert;
```

Comment:
I don't see a scenario for `MULTI_HASH`.

Reply:
The whole change is implicitly tested everywhere in query and change feed anyway. Adding this here mostly to match test coverage and have at least one quick signaly if there is a complete screw-up.